WooCommerce/Classes/ViewRelated/Orders/Order Creation/Synchronizer/ProductInputTransformer.swift

```diff
@@ -41,6 +41,58 @@ struct ProductInputTransformer {
         return order.copy(items: items)
     }
 
+    /// Adds, deletes, or updates order items based on the multiple given product inputs
+    ///
+    /// - Parameters:
+    ///   - input: Array of types of products the synchronizer supports
+    ///   - order: Represents an Order entity.
+    ///   - updateZeroQuantities: When true, items with `.zero` quantities will be updated instead of being deleted.
+    ///
+    /// - Returns: An Order entity.
+    static func updateMultiple(input: [OrderSyncProductInput], on order: Order, updateZeroQuantities: Bool) -> Order {
```

Comment:
I am wondering if we could making this function naming more understandable;` updateMultiple(input ...` sounds incomplete to me. Perhaps `static func updateMultipleItems(with inputs: [OrderSyncProductInput], on order: Order, updateZeroQuantities: Bool) -> Order {`?

Reply:
That sounds definitely better, changes here: https://github.com/woocommerce/woocommerce-ios/pull/9173/commits/9fb9bc51514f708592e4f120c375c19c2ffd21f2